templates/terraform/modules/environment/lambda-db-ops/vairables.tf

```diff
@@ -0,0 +1,18 @@
+variable "project" {
+  description = "The name of the project, mostly for tagging"
+}
+
+variable "environment" {
+  description = "The environment (dev/staging/prod)"
+}
+
+variable "subnet_ids" {
+  description = "Subnet IDs the lambda function is able to connect to inside the VPC"
+  default     = []
+  type        = list(string)
+}
+variable "security_group_ids" {
+  description = "Security groups the lambda function is able to connect to inside the VPC"
+  default     = []
+  type        = list(string)
+}
```

Comment:
This filename has a typo

Reply:
It won't affect anything because terraform just concatenates all the files in the directory regardless of their names but yeah, it should be fixed.